certified-connectors/Talkdesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3046 @@
+              }
+            }
+          },
+          "403": {
+            "description": "Forbidden access.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string",
+                  "description": "Response code.",
+                  "title": "Code"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "Response message.",
+                  "title": "Message"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Response description.",
+                  "title": "Description"
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "User not found.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string",
+                  "description": "Response code.",
+                  "title": "Code"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "Response message.",
+                  "title": "Message"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Response description.",
+                  "title": "Description"
+                }
+              }
+            }
+          },
+          "406": {
+            "description": "Accept format mime type not supported.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string",
+                  "description": "Response code.",
+                  "title": "Code"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "Response message.",
+                  "title": "Message"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "Response description.",
+                  "title": "Description"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get user presence",
+        "description": "Get user presence in Talkdesk.",
+        "operationId": "GetUserPresence",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The user id.",
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "User id",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/ccaas/users/{id}/status": {
+      "put": {
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {}
```

Comment:
@StefanCRT22 Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**